test/integration/maincluster_test.go

```diff
@@ -1183,6 +1189,11 @@ experimental:
   kube2IamSupport:
     enabled: true
   kubeletOpts: '--image-gc-low-threshold 60 --image-gc-high-threshold 70'
+  ipvsProxy:
+    enabled: true
+    scheduler: lc
+    syncPeriod: 900s
```

Comment:
Any reference for properly configuring this setting?
At first glance, a sync period of 15min seems like a bit long - does it mean that a newly added pod/svc IP becomes accessible from the entire cluster after at most 15min?